cmake/FindCMSIS.cmake

```diff
@@ -218,9 +235,17 @@ foreach(COMP ${CMSIS_FIND_COMPONENTS_FAMILIES})
         stm32_get_chip_type(${FAMILY} ${DEVICE} TYPE)
         string(TOLOWER ${DEVICE} DEVICE_L)
         string(TOLOWER ${TYPE} TYPE_L)
+
+        set(STARTUP_NAMES startup_stm32${TYPE_L}.s startup_stm32${TYPE_L}${CORE_Ucm}.s)
+        if(${FAMILY} STREQUAL "MP1")
+            # stm32mp15?dxx.s uses stm32mp15?axx.s
+            # stm32mp15?fxx.s uses stm32mp15?cxx.s
+            string(REPLACE "dxx" "axx" STARTUP_NAMES ${STARTUP_NAMES})
+            string(REPLACE "fxx" "cxx" STARTUP_NAMES ${STARTUP_NAMES})
```

Comment:
this is not what I expected. If I'm correct it will work out of the box once you correct the "type" thing in `MP1.cmake`

Reply:
Understood!